src/shared/components/layout/header/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { MenuOutlined } from '@ant-design/icons';
+import { Drawer, Menu } from 'antd';
+import { Header } from 'antd/lib/layout/layout';
+import { Fragment, useState } from 'react';
+import { useNavigate } from 'react-router';
+
+import Styles from './styles.module.scss';
+
+import LogoSVG from '@/shared/components/svg/LogoSVG';
+import NavigationConst, {
+  defaultKey,
+} from '@/shared/constants/NavigationConst';
+
+export default function AppHeader() {
+  const navigate = useNavigate();
+
+  const [visible, setVisible] = useState(false);
+
+  return (
+    <Header className={Styles.Header}>
+      <LogoSVG width={40} />
+      <MenuOutlined onClick={showDrawer} />
+      <Drawer
```

Comment:
Please implement header for desktop also

Reply:
Header for desktop implemented